src/test/unittests/qt_functions_test.py

```diff
@@ -1,16 +1,21 @@
 import unittest
 from PyQt5 import QtWidgets
 from PyQt5.QtWidgets import QApplication
-import sys
-from src.main.view import qt_functions
+import sys, os
```

Comment:
ez kell ide még akkor?

Reply:
mármint a path hozzáfűzős rész @SzauterAbel

Reply:
Igen, az is kell, sajnos.